extensions/amp-story/1.0/amp-story.js

```diff
@@ -1632,6 +1635,10 @@ export class AmpStory extends AMP.BaseElement {
         actions.execute(this.sidebar_, 'open', /* args */ null,
             /* source */ null, /* caller */ null, /* event */ null,
             ActionTrust.HIGH);
+      } else if (this.sidebar_
+                && !sidebarState
+                && !this.maskElement_.hasAttribute('hidden')) {
```

Comment:
nit: put the operator at the end of the previous line, rather than the beginning of the new line.
also nit: indent this +4 spaces from the indentation level of the line that it's continuing

Reply:
Done.